src/app/shared/custom-components/toolbar/tags/tags.component.html

```diff
@@ -0,0 +1,18 @@
+<div class="container-of-chips">
+  <mat-chip-list>
+    <div class="chips">
```

Comment:
next time make service for getting tags and use ngFor

Reply:
Definitely!